### PR TITLE
Fix Indonesian Translation

### PR DESCRIPTION
**Pull request by ItangSanjana**
Some words are incorrectly and untranslated to Indonesian yet. This pull request is intended to accomplish that.

### CODE REVIEW

File: translations/id.xtb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 <translationbundle lang="id">
-<translation id="1003362845472634045">Email login terkirim</translation>
+<translation id="1003362845472634045">Surel masuk terkirim</translation>
```
Comment (ekoeryanto) [inline translations/id.xtb]: I prefer use *Email* instead of *surel*

File: translations/id.xtb [hunk 2]
```diff
@@ -17,24 +17,24 @@
 <translation id="1268266284642694018">Lebanon</translation>
 <translation id="1294742657282291237">Guyana</translation>
 <translation id="1297168472480293352">Guatemala</translation>
-<translation id="1315809131562953678">Masukan nomor telepon yang valid</translation>
-<translation id="1335716056288807013">Dengan menge-tap Verifikasi, Anda menyatakan bahwa Anda menyetujui <ph name="START_LINK_1" />Persyaratan Layanan<ph name="END_LINK" /> dan <ph name="START_LINK_2" />Kebijakan Privasi<ph name="END_LINK" /> kami. SMS mungkin akan dikirim. Mungkin dikenakan biaya pesan &amp;amp; data.</translation>
+<translation id="1315809131562953678">Masukkan nomor telepon yang valid</translation>
```
Comment (ekoeryanto) [inline translations/id.xtb]: :+1: should be *masukkan* not *masukan*

File: translations/id.xtb [hunk 3]
```diff
@@ -17,24 +17,24 @@
...
 <translation id="1483249363694224399">Sri Lanka</translation>
-<translation id="1507995441600689506">Sekarang Anda dapat login dengan akun baru</translation>
+<translation id="1507995441600689506">Sekarang Anda dapat masuk dengan akun baru</translation>
 <translation id="1575848116712334589">Cad</translation>
 <translation id="1630237004390596779">Kirirmkan kembali kode dalam <ph name="STRING" /></translation>
 <translation id="1653501608290591291">Belarus</translation>
-<translation id="1667376103923043653">Alamat email tidak cocok dengan akun yang ada.</translation>
-<translation id="173576986138855124">Dengan menge-tap "<ph name="PH_1" />", SMS mungkin akan dikirim. Mungkin dikenakan biaya pesan &amp; data.</translation>
+<translation id="1667376103923043653">Alamat surel tidak cocok dengan akun yang ada.</translation>
+<translation id="173576986138855124">Dengan mengetuk "<ph name="PH_1" />", SMS mungkin akan dikirim. Mungkin dikenakan biaya pesan &amp; data.</translation>
```
Comment (ekoeryanto) [inline translations/id.xtb]: :+1:  for *mengetuk*

File: translations/id.xtb [hunk 4]
```diff
@@ -17,24 +17,24 @@
...
 <translation id="137210131544215127">Masukkan sandi Anda</translation>
 <translation id="1393433815796487579">Selesai</translation>
 <translation id="1412449974349991050">Haiti</translation>
-<translation id="144766329299583285">Jika Anda tidak meminta perubahan pada email login, ada kemungkinan seseorang berusaha mengakses akun Anda. Sebaiknya Anda <ph name="START_LINK" />segera mengubah sandi<ph name="END_LINK" />.</translation>
+<translation id="144766329299583285">Jika Anda tidak meminta perubahan pada surel masuk, ada kemungkinan seseorang berusaha mengakses akun Anda. Sebaiknya Anda <ph name="START_LINK" />segera mengubah sandi<ph name="END_LINK" />.</translation>
 <translation id="1449981769524156676">Pulau Heard dan Kepulauan McDonald</translation>
 <translation id="1479035434297790822">Terlalu banyak permintaan akun yang berasal dari alamat IP Anda. Coba beberapa menit lagi.</translation>
 <translation id="1483249363694224399">Sri Lanka</translation>
-<translation id="1507995441600689506">Sekarang Anda dapat login dengan akun baru</translation>
+<translation id="1507995441600689506">Sekarang Anda dapat masuk dengan akun baru</translation>
```
Comment (ekoeryanto) [inline translations/id.xtb]: :+1: for *masuk*

File: translations/id.xtb (hunk 1 above)
Comment (ItangSanjana) [reply translations/id.xtb]: @ekoeryanto yes, I prefer "Email" too but there is "Surel" in Indonesian vocabulary. WHat should we do?  
[sur.el](https://kbbi.kemdikbud.go.id/entri/surel)

**Comment by wti806** [comment]
Hi, guys. I relayed this request to our translation team and here is the feedback from them:

Even though the terms they suggested (e.g. surel, masuk, tautan) are formal Indonesian terms and they are correct, we have many considerations in deciding on terms we are using across our products. We cannot change these terms at the moment, because we use those terms across all our products. Changing them requires a lot of data, planning, and clean-ups. The good news is, we are always doing it! We constantly review our terms and update them with users in mind. One recent term that we update is "menge-tap" into "mengetuk". 

So unfortunately we are not able to approve this change. But if you want to customize the translation, you can always build the library with your own change and use that version. Thank you so much!

**Comment by ItangSanjana** [comment]
> Hi, guys. I relayed this request to our translation team and here is the feedback from them:
> 
> Even though the terms they suggested (e.g. surel, masuk, tautan) are formal Indonesian terms and they are correct, we have many considerations in deciding on terms we are using across our products. We cannot change these terms at the moment, because we use those terms across all our products. Changing them requires a lot of data, planning, and clean-ups. The good news is, we are always doing it! We constantly review our terms and update them with users in mind. One recent term that we update is "menge-tap" into "mengetuk".
> 
> So unfortunately we are not able to approve this change. But if you want to customize the translation, you can always build the library with your own change and use that version. Thank you so much!

Noted that :)

**Pull request closed without merging**